integration/stability/soak_parallel_rm.sh

```diff
@@ -38,6 +39,11 @@ VC_POD_DIR="${VC_POD_DIR:-/var/lib/vc/sbs}"
 # then just set this to a very large number
 MAX_CONTAINERS="${MAX_CONTAINERS:-110}"
 
+if [ "$ID" == debian ]; then
```

Comment:
Hmm, I'd rather we didn't make the link between 'debian == firecracker'. I'd much rather we determine from kata-runtime that firecracker is the installed/default hypervisor.
Are we able to check that from the `kata-env` info maybe? That way the test/disable will continue to work if/when we adjust the CI to run firecracker on other distros as well.